libproto/src/autoimpl.rs

```diff
@@ -38,6 +38,19 @@ pub enum OperateType {
 
 pub const DEFAULT_OPERATE_TYPE: OperateType = OperateType::Broadcast;
 
+pub trait TryFrom<T>
```

Comment:
Since these two traits will be remove in the near future, and before `#![feature(try_from)]` stabilized, I do not want to encourage people to use them, so I did not create a new crate for them.